src/home/WalletHome.tsx

```diff
@@ -102,6 +104,10 @@ function WalletHome() {
   }
 
   const shouldShowCashInBottomSheet = () => {
+    // If user is in a sanctioned country do not show the cash in bottom sheet
+    if (userInSanctionedCountry) {
+      return false
+    }
```

Comment:
Doesn't the CashIn bottom sheet get unmounted when `navigateClearingStack(Screens.SanctionedCountryErrorScreen)` is called?